content/general/concepts/cryptocurrency/cryptocurrency.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'Cryptocurrency'
+Description: 'A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralised networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Blockchain'
+CatalogContent:
+  - 'introduction-to-blockchain'
+  - 'paths/computer-science'
+---
+
+A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralised networks based on blockchain technology, which is a distributed ledger enforced by a network of computers (called nodes) around the world. Cryptocurrencies are not issued or controlled by any central authority, such as the federal bank of a sovereign nation as is the case with fiat money such as the US dollar. Cryptocurrencies are therefore protected from government interference, although this does not mean they cannot be manipulated by other parties.
+
+## Key Concepts
+
+### Blockchain
+
+Blockchain is essentially a set of interconnected “blocks” that form an online ledger that is distributed across computers around the world. Transactions are recorded in blocks that are linked together to form a chain, and these transactions are verified by nodes in the network. A node is a computer on the network running the software client that stores the blockchain data locally. The data is updated as transactions are stored in each block and every copy of the blockchain is updated simultaneously, keeping all the records identical and accurate. As the network relies on consensus mechanisms to verify transactions, it is almost impossible to forge a fraudulent transaction.
+
+Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining, although nowadays the computing power necessary is only achieved through a powerful network of computers known as a “mining pool”.
+
+### Types
+
+Bitcoin is the most popular and widely known cryptocurrency, as well as the most valuable as measured by total market capitalisation. It was created by an individual (or perhaps a group) called Satoshi Nakamoto and introduced via a white paper in 2008. With the success of Bitcoin, thousands of other cryptocurrencies have since been created with different attributes for various functions.
```

Comment:
```suggestion
Bitcoin is the most popular and widely known cryptocurrency, as well as the most valuable as measured by total market capitalization. It was created by an individual (or perhaps a group) called Satoshi Nakamoto and introduced via a white paper in 2008. With the success of Bitcoin, thousands of other cryptocurrencies have since been created with different attributes for various functions.
```